fusor-ember-cli/bower.json

```diff
@@ -13,8 +13,8 @@
     "jquery-csv": "google-code-export/jquery-csv#*",
     "loader.js": "ember-cli/loader.js#3.2.1",
     "qunit": "~1.18.0",
-    "bootstrap-sass": "~3.3.4",
-    "patternfly": "git://github.com/patternfly/patternfly#2.4.0",
+    "patternfly-sass": "2.9.0",
+    "patternfly": "git://github.com/patternfly/patternfly#2.9.0",
```

Comment:
most recent version 3.3.5 but I choose 2.9.0 since this is what is in the branch SATELLITE-6.2.0.